src/Stoa.ts

```diff
@@ -461,8 +471,35 @@ class Stoa extends WebService
             medium: medium.toString(),
             low: low.toString()
         };
+        return data;
+    }
 
-        res.status(200).send(JSON.stringify(data));
+    /**
+     * GET /transaction/estimate/fee/:input/:output/:payload_size
+     *@param req
+     *@param res
+     * Called when a request is received through the `/transaction/estimate/fee/:input/:output/:payload_size` handler
+     *
+     * @returns Returns data fee, transaction size and transaction fee
+     */
+    private async estimateTxFee(req: express.Request, res: express.Response){
+        const input: number = Number(req.params.input)
+        const output: number = Number(req.params.output)
+        const payload_size: number = Number(req.params.payload_size)
+
+        logger.http(`GET /transaction/estimate/fee/${input}/${output}/${payload_size}`)
+        if (!isNaN(input) && !isNaN(output) && !isNaN(payload_size) ){
+            const tx_size: number= Transaction.getEstimatedNumberOfBytes(input, output, payload_size)
+            const dataFee: number = JSBI.toNumber(TxPayloadFee.getFee(payload_size))            
+            const data ={                
+                dataFee,
+                totalFee: this.calculateTransactionFees(tx_size.toString())
+            }
+
+            return res.status(200).send(JSON.stringify(data));
+        }else{
+            return res.status(400).send("Invalid value for parameters");            
+        }
```

Comment:
This does not need to be provided by Stoa.
This can be calculated from the frontend.

Reply:
This feature already exists in the SDK Please use the SDK.